backend/tournesol/views/ratings.py

```diff
@@ -54,9 +58,21 @@ def get_annotated_ratings():
         .order_by("-datetime_lastedit")
     )[:1]
 
+    collective_score = (
+        EntityPollRating.objects.filter(poll=OuterRef("poll"), entity=OuterRef("entity"))
+        .values("tournesol_score")
+    )
+
+    contributor_rating_criteria_score = (
+        ContributorRatingCriteriaScore.objects.filter(contributor_rating=OuterRef("pk"), criteria="largely_recommended")
+        .values("score")
+    )
+
     return ContributorRating.objects.annotate(
         n_comparisons=Subquery(n_comparisons),
         last_compared_at=Subquery(last_compared_at),
+        collective_score_largely_recommended=Subquery(collective_score),
+        contributor_rating_criteria_score=Subquery(contributor_rating_criteria_score)
```

Comment:
Is it a worry for performance that this is now making 5 queries instead of 3? Could we refactor the code a bit such that we actually do only a single annotation for the ordering that is requested?

Reply:
We tested the new query on staging and production and it doesn't look costly. Moreover the execution time of this query set will be further reduced by the presence of the filters on the poll, on the user, and by the pagination (offset / limit).

I think we can keep it as is for now.